src/tiledb/cloud/vector_search/file_ingestion.py

```diff
@@ -0,0 +1,200 @@
+from typing import Dict, List, Mapping, Optional, Sequence
+
+from tiledb.cloud import dag
+from tiledb.cloud.utilities import as_batch
+
+
+def ingest_files_udf(
+    file_dir_uri: str,
+    index_uri: str,
+    file_name: Optional[str] = None,
+    acn: Optional[str] = None,
+    config=None,
+    environment_variables: Optional[Mapping[str, str]] = None,
+    namespace: Optional[str] = None,
+    verbose: bool = False,
+    trace_id: Optional[str] = None,
+    # Index creation params
+    create_index: bool = False,
```

Comment:
What's the reason for having the user check if the index exists or not and settings this parameter vs detecting it internally?

Reply:
This was just to make sure that we don't accidentally write data in an existing dataset.
Removed